xbmc/video/VideoDatabase.cpp

```diff
@@ -182,7 +182,8 @@ void CVideoDatabase::CreateTables()
     "strHdrType text)");
 
   CLog::Log(LOGINFO, "create sets table");
-  m_pDS->exec("CREATE TABLE sets ( idSet integer primary key, strSet text, strOverview text)");
+  m_pDS->exec("CREATE TABLE sets ( idSet integer primary key, strSet text, strOverview text, "
+              "strOriginalSet text)");
```

Comment:
Sorry - I didn't make it clear in the PR description.
In the example at the top - strOriginalSet is 'National Lampoon's Van Wilder Collection' and I want it called 'Van Wilder Collection' - which is strSet.
I save the original name so that if you refresh from scraper (which would only know the original name) you can then tie it in to the new name.